src/raster/r.random.walk/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```

Comment:
Perhaps not as clearly specified for the GitHub submitting procedure, but you don't need to submit a license file here and the license should be GPL >=v2. You do have that in your source code, so I assume just deleting this file will resolve the issue for all.

https://trac.osgeo.org/grass/wiki/Submitting/Python

Reply:
I removed the file.